picard/album.py

```diff
@@ -232,21 +231,21 @@ def _finalize_loading(self, error):
                 for dj in djmix_ars.get(mm["discnumber"], []):
                     mm.add("djmixer", dj)
 
-                if "pregap" in medium_node.children:
+                if "pregap" in medium_node:
                     discpregap = True
                     absolutetracknumber += 1
-                    track = self._finalize_loading_track(medium_node.pregap[0], mm, artists, va, absolutetracknumber, discpregap)
+                    track = self._finalize_loading_track(medium_node['pregap'], mm, artists, va, absolutetracknumber, discpregap)
                     track.metadata['~pregap'] = "1"
 
-                tracklist_node = medium_node.track_list[0]
-                track_count = int(tracklist_node.count)
+                tracklist_node = medium_node['tracks']
+                track_count = int(medium_node['track-count'])
```

Comment:
Just keeping them in case we switch back to strings for these. (I think there were some tickets related to this)